src/components/searchHastag.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="header">
+    <slot name="search"></slot>
```

Comment:
every time you do not need to create a slot component. 
slot components are used only when the HTML elements are dynamic. In searchHastag component slot will not be dynamic @ShivanshSinghnnovative 
This component will be a search component which can be used to show an input filed and emit a function 
 with search teaxt whenever something write in input. @ShivanshSinghnnovative